lumicks/pylake/kymo.py

```diff
@@ -267,25 +276,47 @@ def pixelsize(self):
         list corresponds to the number of scan axes."""
         return [self._calibration.value]
 
-    def _plot(self, channel, axes, adjustment=ColorAdjustment.nothing(), **kwargs):
-        """Plot a kymo for requested color channel(s).
+    @_deprecate_basescan_plot_args
+    def plot(
+        self,
+        channel="rgb",
+        *,
+        adjustment=ColorAdjustment.nothing(),
+        axes=None,
+        image_handle=None,
+        show_title=True,
+        **kwargs,
+    ):
+        """Plot a kymo for the requested color channel(s).
 
         Parameters
         ----------
-        channel : {'red', 'green', 'blue', 'rgb'}
+        channel : {"red", "green", "blue", "rgb"}, optional
             Color channel to plot.
-        axes : mpl.axes.Axes
-            The axes instance in which to plot.
         adjustment : lk.ColorAdjustment
             Color adjustments to apply to the output image.
-
+        axes : mpl.axes.Axes, optional
+            If supplied, the axes instance in which to plot.
+        image_handle : `matplotlib.image.AxesImage` or None
+            Optional image handle which is used to update plots with new data rather than
+            reconstruct them (better for performance).
```

Comment:
should these have the same annotations? maybe `axes : mpl.axes.Axes or None` is nicer and/or I think that's what we use more often? would be good on some rainy day to go through and check for stuff like this

Reply:
👍 I changed it to `matplotlib.*` in the `plot()` functions.